python/paddle/nn/initializer/constant.py

```diff
@@ -35,13 +37,17 @@ class ConstantInitializer(Initializer):
 
     """
 
-    def __init__(self, value=0.0, force_cpu=False):
+    def __init__(self, value: float = 0.0, force_cpu: bool = False) -> None:
         assert value is not None
         super().__init__()
         self._value = value
         self._force_cpu = force_cpu
 
-    def forward(self, var, block=None):
+    def forward(
+        self,
+        var: paddle.Tensor,
+        block: framework.Block | paddle.pir.Block | None = None,
```

Comment:
```suggestion
        block: paddle.pir.Block | None = None,
```

统一下吧，静态图逻辑里不暴露老 IR 的概念，动静统一逻辑里不暴露静态图的概念